### PR TITLE
[KIECLOUD-299] csv generation

### PR DESCRIPTION
**Pull request by tchughesiv**
This generates versioned csv and package files... as well as the operator.yaml and role.yaml files.

`make csv`

/assign @bmozaffa @swati-kale

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: deploy/catalog_resources/community/1.2.1/kiecloud-operator.1.2.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1,216 +1,228 @@
...
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.name
-                      - name: OPERATOR_NAME
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.labels['name']
-                      - name: OPERATOR_UI
-                        value: "true"
+  version: 1.2.1
```
Comment (bmozaffa) [inline deploy/catalog_resources/community/1.2.1/kiecloud-operator.1.2.1.clusterserviceversion.yaml]: Why the changes? Are you committing generated versions?

File: deploy/operator.yaml [hunk 2]
```diff
@@ -4,44 +4,41 @@ metadata:
   name: kie-cloud-operator
 spec:
   replicas: 1
+  selector:
```
Comment (bmozaffa) [inline deploy/operator.yaml]: Why the (non-functional) changes to this (and the role) file?

File: deploy/catalog_resources/community/1.2.1/kiecloud-operator.1.2.1.clusterserviceversion.yaml (hunk 1 above)
Comment (tchughesiv) [reply deploy/catalog_resources/community/1.2.1/kiecloud-operator.1.2.1.clusterserviceversion.yaml]: correct, this is the generated version and should only have this many changes this first time. For the most part, though, it's just a reordering of the fields.

File: deploy/operator.yaml (hunk 2 above)
Comment (tchughesiv) [reply deploy/operator.yaml]: This file is now generated as well and, with this commit, it's just a reordering of some the fields.
